Sources/Sentry/SentryTracer.m

```diff
@@ -48,6 +48,11 @@
 #    import <SentryScreenFrames.h>
 #endif // SENTRY_HAS_UIKIT
 
+#if defined(TEST) || defined(TESTCI)
+#    import "SentryFileManager+Test.h"
```

Comment:
`m`: Having a test header in production code feels weird. Either we have to rename the header or change our code so it doesn't rely on a test header.